services/website-api/src/app.ts

```diff
@@ -155,11 +166,14 @@ const bootServer = async () => {
 
 		logger.error(err);
 
-		res.status(500);
+		res.status(500).send({
+			status: 500,
```

Comment:
isnt that redundant?

Reply:
nope, we only send status code on general error, this way we also send the error message

Reply:
but couldnt you just use res.send instead of the status function